src/app/static/jest.integration.adr.config.js

```diff
@@ -1,4 +1,4 @@
 var config = require('./jest.integration.config');
 
-config.testTimeout = 20000;
+config.testTimeout = 120000;
```

Comment:
I am not complete sure of this integration test behaviour but bumping from 2000 to 12000 seems like a big plus to me.

Reply:
It is, but that seems to be how long it takes :(
I'll try it on 90..